src/aiida/cmdline/commands/cmd_process.py

```diff
@@ -180,14 +180,19 @@ def process_list(
             echo.echo_report(f'Using {percent_load * 100:.0f}% of the available daemon worker slots.')
 
 
-@verdi_process.command('show')
+@verdi_process.command('show', short_help='Show details of processes.')
```

Comment:
Please don't use the `short_help` and just use this as the first line of the docstring.

Reply:
changed